underlay/src/main/resources/config/criteria/cmssynpuf/criteriaselector/conditions/selector.json

```diff
@@ -3,7 +3,6 @@
   "displayName": "Condition",
   "isEnabledForCohorts": true,
   "isEnabledForDataFeatureSets": true,
-  "supportsTemporalQueries": true,
```

Comment:
nit: Can you keep this line and remove the other duplicate? I think it's helpful to have all the flags together (e.g. isEnabled).

Reply:
done